src/main/kotlin/Helper.kt

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner
+
+object Helper {
+    fun inputText(title: String, error: String): String {
+        val scanner = Scanner(System.`in`)
+
+        while (true) {
+            println(title)
+            val text = scanner.nextLine().trim()
+
+            if (text.length == 0) {
```

Comment:
```suggestion
            if (text.length.isEmpty()) {
```
Проверку лучше заменить на `isEmpty()` - так данный код будет читабельнее